### ISSUE TITLE
ec2_instance instance watcher repeats instance information 

### ISSUE THREAD

**Issue by shrikant0013** [Netflix/security_monkey#798]
ec2_instance instance watcher repeats instance information in the list due to bug in the code. 
This leads to incorrect instance count and information in SecurityMonkey database.
Below code could be the issue. Instances list is iterated twice

```python
                for reservation in reservations:
                    instances = reservation.get('Instances')
                    for instance in instances:
                        instances = reservation.get('Instances')
                        for instance in instances:
                            name = None
```

**Comment by monkeysecurity** [comment]
Should be fixed in #799.

**Comment by monkeysecurity** [comment]
PR merged. Closing issue.
